generators/app/index.js

```diff
@@ -93,29 +96,51 @@ module.exports = class extends BaseGenerator {
                     .split('.')
                     .slice(0, -1)
                     .join('.')
-                    .match(/\w*$/i);
+                    .match(/\w*$/i)[0];
```

Comment:
Why take only the first one?

Reply:
Because the result of the regex exécution is an object :
```javascript
[
  'JhiOrder',
  index: 37,
  input: 'src/main/java/test/kafka/msng/domain/JhiOrder',
  groups: undefined
]
```
But only the entity name is useful.